releasetasks/templates/desktop/funsize_image.yml.tmpl

```diff
@@ -28,12 +28,12 @@
             onExitStatus:
                 retry: [-1, 1, 255]
             env:
-                HASH: "{{ revision }}"
+                HASH: "{{ mozharness_changeset }}"
```

Comment:
do we want to leave these changes in?

Reply:
This change is not exactly related, but it does the right thing.